.github/workflows/cpdb_build.yml

```diff
@@ -17,27 +17,24 @@ jobs:
     container: 
       image: nycplanning/build-geosupport:latest
       env:
-        BUILD_ENGINE: postgresql://postgres:postgres@postgis:5432/postgres
+        BUILD_ENGINE_DATABASE: ${{ secrets.SQL_ENGINE_EDM_DATA_SERVER }}/db-cpdb
+        BUILD_ENGINE_SCHEMA: build_${{ github.event.pull_request.number || github.ref_name }}
         EDM_DATA: ${{ secrets.EDM_DATA }}
         AWS_S3_ENDPOINT: ${{ secrets.AWS_S3_ENDPOINT }}
         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-    services:
-      postgis:
-        image: postgis/postgis:15-3.3-alpine
-        env:
-          POSTGRES_PASSWORD: postgres
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-
-        ports:
-          - 5432:5432
     steps:
       - uses: actions/checkout@v3
 
+      - name: Set build environment variables
+        run: |
+          # # replace dashes with underscores to create a valid postgres schema name
+          BUILD_ENGINE_SCHEMA=$(echo ${BUILD_ENGINE_SCHEMA} | tr - _)
+          echo "BUILD_ENGINE_SCHEMA=$BUILD_ENGINE_SCHEMA" >> "$GITHUB_ENV"
+          # set postgres schema search path to prioritize BUILD_ENGINE_SCHEMA
+          BUILD_ENGINE=${BUILD_ENGINE_DATABASE}?options=--search_path%3D${BUILD_ENGINE_SCHEMA},public
+          echo "BUILD_ENGINE=$BUILD_ENGINE" >> "$GITHUB_ENV"
+
```

Comment:
`PostgresClient` would handle this if the CPDB build ran SQL queries via python

Reply:
Oh well. If this is being used in a couple build files, maybe make this a bash function or script?

Reply:
true! definitely trying to deprecate this approach too per [this thread below](https://github.com/NYCPlanning/data-engineering/pull/269#discussion_r1349316650)

would love to save ur idea for the follow-up PR that expands the use of the build DB to other products